.ado/publish.yml

```diff
@@ -87,301 +87,368 @@ variables:
+              Start-Sleep -Seconds 60
+            displayName: Stop pipeline if latest commit message contains ***NO_CI***
+            condition: and(${{ parameters.stopOnNoCI }}, contains(variables['Build.SourceVersionMessage'], '***NO_CI***'))
+
+          - template: .ado/templates/checkout-full.yml@self
+            parameters:
+              persistCredentials: false # We're going to use rnbot's git creds to publish
+
+          - powershell: gci env:/BUILD_*
+            displayName: Show build information  
+
+          - template: .ado/templates/prepare-js-env.yml@self
+
+          - template: .ado/templates/run-compliance-prebuild.yml@self
+
+          - template: .ado/templates/configure-git.yml@self
+
+          - script: |
+              echo ##vso[task.setvariable variable=SkipNpmPublishArgs]--no-publish
+            displayName: Enable No-Publish
+            condition: ${{ parameters.skipNpmPublish }}
+
+          - script: |
+              echo ##vso[task.setvariable variable=SkipGitPushPublishArgs]--no-push
+            displayName: Enable No-Publish
+            condition: ${{ parameters.skipGitPush }}
+
+          - script: if not exist %USERPROFILE%\AppData\Roaming\npm (mkdir %USERPROFILE%\AppData\Roaming\npm)
+            displayName: Fix missing npm config
+
+          - pwsh: |
+              npx beachball check --verbose 2>&1 | Tee-Object -Variable beachballOutput
+              $beachballOutput | Where-Object { $_ -match "ERROR: *"} | ForEach { Write-Host "##vso[task.logissue type=error]$_" }
+            displayName: Beachball Check
+
+          - script: npx beachball publish $(SkipNpmPublishArgs) $(SkipGitPushPublishArgs) --branch origin/$(Build.SourceBranchName) -n $(npmAuthToken) -yes --bump-deps --verbose --access public --message "applying package updates ***NO_CI***"
+            displayName: Beachball Publish
+
+          # Beachball reverts to local state after publish, but we want the updates it added
+          - script: git pull origin ${{ variables['Build.SourceBranchName'] }}
+            displayName: git pull
+
+          - script: npx @rnw-scripts/create-github-releases --yes --authToken $(githubAuthToken)
+            displayName: Create GitHub Releases (New Canary Version)
+            condition: and(succeeded(), ${{ not(parameters.skipGitPush) }}, ${{ eq(variables['Build.SourceBranchName'], 'main') }} )
+
+          - script: npx --yes @rnw-scripts/create-github-releases@latest --yes --authToken $(githubAuthToken)
+            displayName: Create GitHub Releases (New Stable Version)
+            condition: and(succeeded(), ${{ not(parameters.skipGitPush) }}, ${{ ne(variables['Build.SourceBranchName'], 'main') }} )
+
+          - template: .ado/templates/set-version-vars.yml@self
+            parameters:
+              buildEnvironment: Continuous
+
+          - task: AzureArtifacts.manifest-generator-task.manifest-generator-task.ManifestGeneratorTask@0
```

Comment:
YOu can lave the sbom tasks out. Just need to add an 
outputs section to the job....

Reply:
else other required checks don't run either.

Reply:
Yeah, but there is no output because beachball doesn't create any artifact to upload. I checked with support and they said what I was doing was fine.